index.bs

```diff
@@ -3613,16 +3613,18 @@ the process by which the [=CBOR=] [=authenticator extension output=] can be used
 
 ## <dfn>Authenticator extension processing</dfn> ## {#sctn-authenticator-extension-processing}
 
-The [=CBOR=] [=authenticator extension input=] value of each processed
-[=authenticator extension=] is included in the extensions data part of the authenticator request. This part is a CBOR map, with
-[=CBOR=] [=extension identifier=] values as keys, and the [=CBOR=] [=authenticator extension input=] value of each extension as
-the value.
+The [=CBOR=] <dfn>authenticator extension input</dfn> value of each processed [=authenticator extension=] is included in the
+|extensions| parameter of the [=authenticatorMakeCredential=] and [=authenticatorGetAssertion=] operations. The |extensions|
+parameter is a [=CBOR=] map where each key is an [=extension identifier=] and the corresponding value is the [=authenticator
+extension input=] for that extension.
 
-Likewise, the extension output is represented in the [=authenticator data=] as a CBOR map with [=CBOR=] [=extension
-identifiers=] as keys, and the [=CBOR=] <dfn>authenticator extension output</dfn> value of each extension as the value.
+Likewise, the <dfn>authenticator extension output</dfn> value of each processed [=authenticator extension=] is represented by one
```

Comment:
This is just wrong.  The "authenticator extension output" is the value of the key/value pair.  You're using the term in a way that makes it sound like it's both the key and the value.  Please revert this change.

Reply:
Ah yes, sorry.